drivers/dma/dma_stm32.c

```diff
@@ -132,15 +132,15 @@ static void dma_stm32_irq_handler(const struct device *dev, uint32_t id)
 
 #ifdef CONFIG_DMA_STM32_SHARED_IRQS
 
-#define HANDLE_IRQS(index)						       \
```

Comment:
We're not chaning anything in this file, so I we should remove all changes to `dma_stm32.c` in this PR. This is to keep the PR clean.

Reply:
restored original file